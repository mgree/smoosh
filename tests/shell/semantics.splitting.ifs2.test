arg_len() {
    echo "$#"
}

IFS=":"
FOO=":"

[ "$(arg_len $FOO)" -eq 1 ] && echo OK 1
[ "$(arg_len $FOO $FOO)" -eq 2 ] && echo OK 2a
[ "$(arg_len $FOO$FOO)" -eq 2 ] && echo OK 2b
[ "$(arg_len $FOO $FOO $FOO)" -eq 3 ] && echo OK 3a
[ "$(arg_len $FOO$FOO$FOO)" -eq 3 ] && echo OK 3b
