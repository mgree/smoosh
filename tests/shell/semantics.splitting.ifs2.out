OK 1
OK 2a
OK 2b
OK 3a
OK 3b
